incl/lib/connection.php

```diff
@@ -1,18 +1,17 @@
 <?php
+
 //error_reporting(0);
 include dirname(__FILE__)."/../../config/connection.php";
 @header('Content-Type: text/html; charset=utf-8');
-if(!isset($port))
-	$port = 3306;
+if (!isset($port)) {
+    $port = 3306;
+}
```

Comment:
Adding brackets into a single line of code is completely unnecessary.

Reply:
> Adding brackets into a single line of code is completely unnecessary.

Completely agree